app/controllers/stripe_active_cards_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
     end
   rescue Stripe::InvalidRequestError
     DataDogStatsClient.increment("stripe.errors")
+    DataDogStatsClient.increment("stripe.errors.new_subscription")
```

Comment:
The original error from Timber is searching the logs for _"Stripe New Subscription Failure"_. I searched as far back as possible to see where exactly that error would be raised and I didn't it at all. My best guess was it happens here.